test/Dapr.Actors.AspNetCore.IntegrationTest.App/Dapr - Backup.Actors.AspNetCore.IntegrationTest.App.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Can you please undo this and other similar files? I don't think we need extra copies of the test project files